front/src/common/Map/const.ts

```diff
@@ -105,5 +105,49 @@ export const SEARCH_URL = `/editoast/search`;
 
 export type searchPayloadType = {
   object: string;
-  query: (string | any[])[];
+  query: (string | boolean | any[])[];
 };
+
+export const signalAspects = [
+  { label: 'Feu Vert' },
```

Comment:
Can you delete all the labels ? It's best to have directly an array of strings I think
```suggestion
  'Feu Vert',
```

Reply:
Done